src/test/java/org/fundacionjala/virtualassistant/openai/OpenAIClientTest.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.virtualassistant.openai;
+
+
+import com.theokanning.openai.completion.CompletionChoice;
+import com.theokanning.openai.completion.CompletionResult;
+import com.theokanning.openai.service.OpenAiService;
+import org.fundacionjala.virtualassistant.clients.openai.OpenAIClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class OpenAIClientTest {
+
+    private OpenAIClient client;
+    private OpenAiService service;
+
+    @BeforeEach
+    void setUp() {
+        service = mock(OpenAiService.class);
+        client = new OpenAIClient(service);
+    }
+
+
+    @Test
+    public void chat() {
+        CompletionChoice choice = new CompletionChoice();
+        choice.setText("Hello world");
+        CompletionResult completion = new CompletionResult();
+        completion.setChoices(Collections.singletonList(choice));
+
+        when(service.createCompletion(any())).thenReturn(completion);
+        String result = client.chat("Hi");
```

Comment:
Why the client is returning an string saying "Chat result is not null"?

Reply:
Well, since it is a request to a chat bot, the response will never be the same and an assertion cannot be placed for this reason, so in this test I send the request and receive a response that is expected to be non-null

Reply:
but in that case you need to verify that the string is not null, and return a true or false..

Reply:
changes already applied